### PR TITLE
Wallet page

### LINKED ISSUE #29: Wallet page
Display NFT assets using Keyp NFT API

See https://github.com/UseKeyp/keyp-app/pull/324

image

### PR DESCRIPTION
**Pull request by alekseevaiana**
closes #29

### CODE REVIEW

**Comment by alekseevaiana** [comment]
## Screenshots

Screenshot 2023-07-13 at 3 37 31 PM

Screenshot 2023-07-13 at 3 37 56 PM

**Comment by alekseevaiana** [comment]
This page sometimes freezes. But cannot understand why. When I try to logout I got an error on account page as well.
So will work on QA issue now to fix account page. And something could be wrong with my access token when balance fetching.

File: components/NFTsList.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+const NFTsList = () => {
+  const [nftsList, setNftsList] = useState<Partial<NFT>[]>([]);
+  const [activeTooltip, setActiveTooltip] = useState<number | null>(null);
+
+  const { data: session } = useSession();
+
+  const fetchNFTs = () => {
+    const ACCESS_TOKEN = process.env.NEXT_PUBLIC_ACCESS_TOKEN;
+    // const ACCESS_TOKEN = session?.user.accessToken;
+    const userId = "0x3ab208D3CE512F2ac0Aa821Eecf2B816A96799B0"; // account with nfts
```
Comment (pi0neerpat) [inline components/NFTsList.tsx:38]: Revert back to the session token

File: components/NFTsList.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    // const userId = session?.user.id;
+    const options = {
+      headers: {
+        Authorization: `Bearer ${ACCESS_TOKEN}`,
+      },
+    };
+
+    // const getNft = `http://api.usekeyp.com/v1/nfts/${userId}`;
+    const getNft = `http://localhost:4001/v1/nfts/${userId}`;
```
Comment (pi0neerpat) [inline components/NFTsList.tsx:47]: Revert back to the keyp api

**Review by pi0neerpat — APPROVED** [review]

**Comment by alekseevaiana** [comment]
@pi0neerpat could you remind me if I can merge this branch into dev or we are waiting when `/nfts/:userId` API will be in a production?

**Pull request merged**